docs/guide/index.qmd

```diff
@@ -4,7 +4,10 @@ description: "A guide on using `check-datapackage` to check the compliance of yo
 listing:
     contents: .
     type: grid
+    grid-columns: 1
+    grid-item-border: false
```

Comment:
Personally, I prefer including the borders. Without them, each items seems less like a separate clickable heading to me and instead blends together with the other headings on the page:

Without border (current):
<img width="923" height="774" alt="Image" src="https://github.com/user-attachments/assets/dc5a54e2-81d6-49a7-a2a5-7b3947f32886" />

With border:
<img width="923" height="774" alt="Image" src="https://github.com/user-attachments/assets/b54701b5-4796-4b39-ac72-18a1430e46ec" />